src/test/java/dog/pawbook/testutil/ProgramBuilder.java

```diff
@@ -0,0 +1,69 @@
+package dog.pawbook.testutil;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import dog.pawbook.model.managedentity.Name;
+import dog.pawbook.model.managedentity.program.Program;
+import dog.pawbook.model.managedentity.program.Session;
+import dog.pawbook.model.managedentity.tag.Tag;
+import dog.pawbook.model.util.SampleDataUtil;
+
+/**
+ * A utility class to help with building Owner objects.
+ */
+public class ProgramBuilder {
```

Comment:
Similar to the builders as well, using OOP to adhere to DRY principle is better since we have no test code for test code, else this will result in some infinite recursion.